content/css/jenkins.css

```
@@ -1770,11 +1770,17 @@ table.syntax > tbody > tr > th {
   border: 1px solid #d3d3d3;
   border-radius: .25rem;
 }
-/* add border to tables */
-.tableblock.halign-left.valign-top { 
-  border: 1px solid #dee2e6; padding: 10px; 
-}
 
 .grid-container .card .content {
   padding: 15px;
 }
+
+/* add border to tables */
+.grid-all, .grid-all th, .grid-all td { 
+  border: 1px solid #dee2e6; 
+  padding: .75rem; 
```

Remove trailing white space
```suggestion
.grid-all, .grid-all th, .grid-all td {
  border: 1px solid #dee2e6;
  padding: .75rem;
```